docs/UserGuide.md

```diff
@@ -634,10 +634,16 @@ Format: `clear`
 
 ### Undo the last command: `undo`
 
-Undoes the last command executed and restores the Teacher's Pet to the previous state.
+Undoes the last command executed by reverting any recent change in data within the Teacher's Pet.
 
 Format: `undo`
 
+<div markdown="span" class="alert alert-success">💡 **Tip:** Undo command will not change any visual display
```

Comment:
> Should this be a Note instead of a Tip ? also correct me if I'm wrong but I think we will be removing callouts (tips, warnings) and just have Note and Caution. (ps. we will be renaming "Callouts" to "Tips" or something more easily understood by reader right) @seanflyyy @cadencjk @gnahzeus

I think it does make sense to be a note as this is something the writer should be aware of. I think we can add this block for full explanation purpose and convert to the new forms when we have decided to do so.